helper/useragent/useragent.go

```diff
@@ -75,10 +75,18 @@ func AgentAutoAuthString() string {
 		versionFunc(), projectURL, rt)
 }
 
-// ProxyString returns the consistent user-agent string for Vault Proxy API Proxying.
+// ProxyString returns the consistent user-agent string for Vault Proxy.
 //
-// e.g. Vault Proxy API Proxy/0.10.4 (+https://www.vaultproject.io/; go1.10.1)
+// e.g. Vault Proxy/0.10.4 (+https://www.vaultproject.io/; go1.10.1)
 func ProxyString() string {
+	return fmt.Sprintf("Vault Proxy/%s (+%s; %s)",
+		versionFunc(), projectURL, rt)
+}
+
+// ProxyAPIProxyString returns the consistent user-agent string for Vault Proxy API Proxying.
+//
```

Comment:
What do you think of ditching these extra comment-spacer lines that are in this file for funcs, I haven't really seen us use them in godocs?

Reply:
Personally, I kind of like them for this specific use-case, as it leaves a space before the description and the example. I'm happy to remove them it other folks feel strongly, though. I was mostly following precedent, and the precedent looked nice enough to me for me to continue it, hehe